jso/apis/src/main/java/org/teavm/jso/dom/html/HTMLInputElement.java

```diff
@@ -65,6 +65,9 @@
 
     @JSProperty
     void setValue(String value);
+    
+    @JSProperty
```

Comment:
Please, remove `@JSProperty` annotations from `setCustomValidity` methods, since they are methods, not properties.

Reply:
Thanks for the feedback. Got it. Will update immediately.